### PR TITLE
Update for Fastify v5

### PR DESCRIPTION
**Pull request by synapse**
Ref: https://github.com/fastify/fastify/issues/5116

⚠️ There's already a `next` branch here

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

CC @simoneb

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,6 +17,6 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v3
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.1.1
```
Comment (simoneb) [inline .github/workflows/ci.yml:20]: oh, we've got 4.1.1 now? what do we do with all the others which are at 4.1.0?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/busboy",
-  "version": "2.1.0",
+  "version": "2.1.1",
```
Comment (simoneb) [inline package.json:3]: why the bump?

File: package.json [hunk 3]
```diff
@@ -36,15 +36,15 @@
     "node": ">=14"
```
Comment (simoneb) [inline package.json:36]: we should probably change this too if we're now supporting node 16+

File: test/decode-text.test.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const { test } = require('tap')
+const decodeText = require('../lib/utils/decodeText')
+
+test('decodeText', t => {
```
Comment (simoneb) [inline test/decode-text.test.js:6]: why this new test?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (gurgunday) [reply .github/workflows/ci.yml:20]: Nope it should be 4.1.0

But we have the same situation with v4.0.0 and v4.1.0 and the only difference is the BlueOak license

So for now let’s try to use the latest versions but it’s fine if some repos don’t

File: package.json (hunk 3 above)
Comment (gurgunday) [reply package.json:36]: We remove engines fields now for maintenance purposes so that should be removed

**Review by synapse — COMMENTED** [review]
Fixed typo

**Review by synapse — COMMENTED** [review]
removed engine field

File: package.json (hunk 2 above)
Comment (synapse) [reply package.json]: The bump came from the current/old `next` branch

**Review by gurgunday — APPROVED** [review]
lgtm

**Pull request merged**